app.rb

```diff
@@ -0,0 +1,66 @@
+require_relative 'book'
+require_relative 'label'
+
+class App
+  def initialize
+    @books = []
+    @albums = []
+    @games = []
+    @genres = []
+    @labels = []
+    @authors = []
+  end
+
+  def select(action)
+    case action
+    when 1
+      list_all_books
+
+    when 5
+      list_all_labels
+
+    when 7
+      add_a_book
+    end
+  end
+
+  def add_a_book
+    puts "\nPublisher\n"
+    publisher = gets.chomp
+    puts "\nCover state\n"
+    cover = gets.chomp
+    puts "\nPublish date\n"
+    date = gets.chomp
+    book = Book.new(publisher, cover, date)
+    @books.push(book)
+    puts "\nWould you like to add a label?\n"
```

Comment:
@Daphineatim optionally give the client options such as yes or no, this prevents invalid inputs.

Reply:
Thank you for the feedback.